src/web/components/elements/YoutubeBlockComponent.tsx

```diff
@@ -125,17 +127,42 @@ export const YoutubeBlockComponent = ({
         );
     }
 
+    const ophanTracking = (trackingEvent: string) => {
```

Comment:
As before, take it or leave it based on context, but is there a more imperative name here to represent what the function _does_?